src/response/error.rs

```diff
@@ -147,10 +163,12 @@ pub enum ErrorSource {
 
 impl From<String> for MomentoError {
```

Comment:
where does this get used? It seems sketchy to assume that anytime someone wants to convert a String to a MomentoError that it would always be a `BadRequest`? I'm not sure it even makes sense to me to implement `From<String>` for MomentoError, but if it did make sense I would expect it to map to like an UnknownError.

Reply:
not used anywhere apparently, will remove